src/containers/ArticlePage/components/ArticleHero.tsx

```diff
@@ -60,24 +76,16 @@ const ArticleHero = ({
     )}
     <OneColumn>
       <div className="c-hero__content">
-        <section>{subject && <Breadcrumb items={breadcrumbItems} />}</section>
+        <section>
+          {subject && (
+            <Breadcrumb items={breadcrumbItems} invertedStyle={false}>
+              <></>
```

Comment:
Er det meininga å ha en tom fragment her?

Reply:
Breadcrumb krever children i frontend-packages, så jeg passerer inn en Fragment for å gjøre TypeScript fornøyd.